dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -84,6 +84,26 @@
 			interrupts = <5 0>;
 			status = "disabled";
 		};
+
+		spi: spi@50020300 {
+			compatible = "renesas,smartbond-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x50020300 0x100>;
+			periph-clock-config = <0x20>;
```

Comment:
for future: this should use `clocks` property

Reply:
There's a coming PR with clock config. We'll keep in mind to update the peripherals